bioio_nd2/reader.py

```diff
@@ -130,3 +145,17 @@ def ome_metadata(self) -> OME:
                 with nd2.ND2File(f) as rdr:
                     return rdr.ome_metadata()
         raise NotImplementedError()
+
+    @property
+    def standard_metadata(self) -> StandardMetadata:
+        """
+        Return the standard metadata for this reader, updating specific fields.
+
+        This implementation calls the base reader’s standard_metadata property
+        via super() and then assigns the new values.
+        """
+        metadata = super().standard_metadata
+        metadata.binning = self.binning
+        metadata.position_index = self.current_scene_index
```

Comment:
`        metadata.position_index = self.current_scene_index`

If you haven't yet, triple check this is true using a viewer (like Zen but whichever viewer readers Nikon images) Antoine or Chris F. can help with this.